.changeset/cute-files-pay.md

```
@@ -0,0 +1,12 @@
+---
+"task-master-ai": minor
+---
+
+Add compact mode --compact / -c flag to the `tm list` CLI command
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)